### DISCUSSION TITLE
New Terminal Vscode tidak sama dengan apa yang ditunjukan di video. saat saya memasukan code conda activate capstone-flask, ada tulisan 'conda' is not recognized as an internal or external command

### DISCUSSION THREAD

**Discussion by MatthewBradleyLo** [teamalgoritma/community#660] [Q&A] [answered]
![Screenshot (14)](https://github.com/teamalgoritma/community/assets/133225102/5c26b787-7305-4de4-a24d-3874a4c958df)

**Comment by nadyadtm** [comment, marked as answer]
Dear Pak @MatthewBradleyLo 

Agar cmd conda bisa dijalankan dalam VS Code, bapak perlu memasukan path Anaconda ke dalam Environment Variable. Caranya adalah sebagai berikut
1. tekan **Windows + R**, lalu masukkan **sysdm.cpl**
2. Lalu kemudian masuk ke dalam environment variable dengan memilih tab **Advance** -> **Environment Variable**
![image](https://github.com/teamalgoritma/community/assets/15353477/79e042aa-ae76-4be4-8ac1-33558691aab0)
3. Kemudian pilih **Path** yang ada di bagian **User Variables**, lalu klik **Edit**
![image](https://github.com/teamalgoritma/community/assets/15353477/186210a6-1414-4e94-adee-1a755d26cc87)
4. Silakan tambahkan dua path berikut ini, untuk bagian LENOVOnya bisa sesuaikan dengan folder user Bapak, 
contoh : C:\Users\Username\anaconda3\Scripts,  C:\Users\Username\anaconda3\
Untuk menambahkannya dapat klik **New**
![image](https://github.com/teamalgoritma/community/assets/15353477/ce4045ae-072b-4fd6-9da4-94a0bdb0ded0)
5. Jangan lupa untuk klik **Ok**

Lalu silakan coba jalankan lagi. Semoga membantu.

**Reply by MatthewBradleyLo** [reply]
tekan windows + R nya di tab mana? soalnya saya windows+R di VS code gk sama diannya

**Reply by nadyadtm** [reply]
Tidak di VS Codenya pak, atau bapak bisa buka menu "Run" yang ada di menu windowsnya. Bila masih mengalami kendala silakan untuk mengikuti step yang ada disini https://www.partitionwizard.com/partitionmanager/conda-is-not-recognized.html

Tambahan, ada di bagian Fix 3: Add the Conda Path Manually

**Comment by nadyadtm** [comment]
Halo @MatthewBradleyLo, untuk menjaga agar forum diskusi berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah lebih dari 7 hari respon terakhir. Semoga dapat dipahami

**Comment by MatthewBradleyLo** [comment]
oke kak
